cmd/kubernetes.go

```diff
@@ -192,17 +196,17 @@ var execCredentialCmd = &cobra.Command{
 				Status: &clientauth.ExecCredentialStatus{
 					ClientKeyData:         string(clientKeyData),
 					ClientCertificateData: string(clientCertificateData),
-					ExpirationTimestamp:   &metav1.Time{Time: time.Now().Add(time.Hour)},
+					ExpirationTimestamp:   &metav1.Time{Time: expirationTime},
 				},
 			}
 
 			if err := cacheKubeConfig(clusterID, execCredential); err != nil {
 				fmt.Fprintln(os.Stderr, err)
 			}
-		}
-		if execCredential == nil {
-			fmt.Println("Error: Could not retrieve kubeconfig from provider for account: ", account)
-			return
+
+			if execCredential == nil {
```

Comment:
why do you check `execCredential == nil`? I see that line 191 says `execCredential = &clientauth.ExecCredential`

Reply:
it might be my bad! I think it can be removed!

Reply:
@bkircher If @itakouna did so, could you remove it please?

Reply:
Ah, sure. Good one. Didn't see that one.